components/data_comps/datm/bld/namelist_files/namelist_definition_datm.xml

```diff
@@ -4,32 +4,93 @@
 
 <namelist_definition>
 
-<!-- =========================================  -->
-<!--- DATM env settings                         -->
-<!-- =========================================  -->
+<!-- ========================================================================================  -->
+<!-- Stream file variables                                                                     -->
+<!-- ========================================================================================  -->
 
-<entry id="DATM_MODE"
-type="char*256"
-category="datm_setting"
-group="datm_env_settings" 
-valid_values="CLM_QIAN,CLM1PT,CLMCRUNCEP,CLMCRUNCEP_V5,CLMGSWP3,CPLHIST3HrWx,CORE2_NYF,CORE2_IAF,COPYALL_NPS_v1,COPYALL_CORE2_v1,WW3,NLDAS">
-Data Model mode to process forcing data
+<entry id="streamslist"
+type="char*1024(100)"
+category="streams"
+group="streams_file">
+List of streams used for the given datm_mode.
 </entry>
 
-<entry id="DATM_PRESAERO"
-type="char*256"
-category="datm_setting"
-group="datm_env_settings"
-valid_values="none,pt1_pt1,clim_2000,clim_1850,trans_1850-2000,rcp2.6,rcp4.5,rcp6.0,rcp8.5">
-Prescribed aerosol mode
+<entry id="strm_domdir"
```

Comment:
I have a philosophical issue with the format of this xml file.  The issue is in the distinction between attributes and entries:  this file incorrectly handles everything as attributes.   It would be more correct and consistent with other xml in cime to have:
```
    <entry id="strm_domdir">
        <type>char*4096</type>
        <category>streams</category>
        <group>streams_file</group>
        <desc>Stream domain file directory</desc>
    </entry>